### PR TITLE
Adjusting upgrade docs from 6 -> 7 (removing)

### PR DESCRIPTION
**Pull request by asettle**
We've had an issue (or two) with customers erroneously upgrading to version 7 when they can't. Deepsea won't allow the upgrade anymore and thus these docs are partially redundant. I'm open to suggestion if people have a better way to point our customers in the right direction.

Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

**Comment by asettle** [comment]
It's complaining about two hyphens in the middle of the docs - I can simply remove them if we agree to the change, but otherwise, it's fine.

**Review by tbazant — APPROVED** [review]
Looks good (if it is true)

File: xml/admin_ceph_upgrade.xml [hunk 1]
```diff
@@ -50,7 +53,7 @@
    supported. First, you must upgrade to the latest version of &productname;
    &prevproductnumber;, and then follow the steps in this chapter.
   </para>
- </important>
+<!---</important>
```
Comment (tbazant) [inline xml/admin_ceph_upgrade.xml:56]:
```suggestion
<!-- </important>
```

**Comment by tbazant** [comment]
> Looks good (if it is true)

**Pull request merged**